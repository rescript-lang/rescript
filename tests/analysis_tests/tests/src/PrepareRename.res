let x = 12
//  ^pre

let foo = (~xx) => xx + 1
//                 ^pre

