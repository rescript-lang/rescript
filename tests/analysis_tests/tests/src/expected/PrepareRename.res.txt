PrepareRename src/PrepareRename.res 0:4
{
    "range": {"start": {"line": 0, "character": 4}, "end": {"line": 0, "character": 5}},
    "placeholder": "x"
  }

PrepareRename src/PrepareRename.res 3:19
{
    "range": {"start": {"line": 3, "character": 19}, "end": {"line": 3, "character": 21}},
    "placeholder": "xx"
  }

